flyteplugins/go/tasks/pluginmachinery/k8s/plugin.go

```diff
@@ -30,6 +30,16 @@ type PluginEntry struct {
 	CustomKubeClient func(ctx context.Context) (pluginsCore.KubeClient, error)
 }
 
+type ErrorAggregationStrategy int
+
+const (
+	// Single error file from a single container
+	DefaultErrorAggregationStrategy ErrorAggregationStrategy = iota
+
+	// Earliest error from potentially multiple error files
+	EarliestErrorAggregationStrategy
+)
```

Comment:
Nit (non blocking):
In the RFC we currently propose:

```go
type ErrorAggregationStrategy int

const (
	// Single error file from a single container
	Default ErrorAggregationStrategy = iota

	// Earliest error from potentially multiple error files
	Earliest
```

I personally prefer to not include the type in the variable name.

Reply:
I'm having difficulty adapting to this convention, because Golang's 'poor' support for enums. If we don't have the type in the variable name, the use site will look like `k8s.Earliest` instead of `k8s.EarliestErrorAggregationStrategy` :(

Reply:
i'm okay either way.

Reply:
I didn't consider this would lead to `k8s.Earliest`, enums in go are truly sad, let's do it as you proposed 👍